kanidmd/Dockerfile

```diff
@@ -1,22 +1,21 @@
 FROM opensuse/tumbleweed:latest
 MAINTAINER william@blackhats.net.au
 
-EXPOSE 8080
-
 COPY . /home/kanidm/
 
 WORKDIR /home/kanidm/
 
 RUN zypper install -y timezone cargo rust gcc sqlite3-devel libopenssl-devel && \
-    RUSTC_BOOTSTRAP=1 cargo build --release && \
-    zypper rm -u -y cargo rust gcc && \
-    zypper clean
+    RUSTC_BOOTSTRAP=1 cargo build --release
 
-RUN cd /etc && \
-    ln -sf ../usr/share/zoneinfo/Australia/Brisbane localtime
+FROM opensuse/tumbleweed:latest
+
+EXPOSE 8080
+WORKDIR /
+COPY --from=0 /home/kanidm/target/release/kanidmd /sbin/
 
 VOLUME /data
 
 ENV RUST_BACKTRACE 1
-CMD ["/home/kanidm/target/release/kanidmd", "server", "-D", "/data/kanidm.db",  "-C", "/data/ca.pem", "-c", "/data/cert.pem", "-k", "/data/key.pem", "--bindaddr", "0.0.0.0:8080"]
+CMD ["/sbin/kanidmd", "server", "-D", "/data/kanidm.db",  "-C", "/data/ca.pem", "-c", "/data/cert.pem", "-k", "/data/key.pem", "--bindaddr", "0.0.0.0:8080"]
```

Comment:
looks like you have the structopt defaults, then the docker defaults, then the override in the getting started, why not remove one or two of em?

Reply:
The plan is to cemove the need for a user to supply the cli - the issue is the -d param at until I impl #127 for domain (saying this, domain currently doesn't do anything ..... I think?)